src/journal/journalctl.c

```diff
@@ -683,8 +683,9 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_VERIFY_KEY:
                         arg_action = ACTION_VERIFY;
-                        arg_verify_key = optarg;
+                        arg_verify_key = strdup(optarg);
```

Comment:
> oom check is necessary.

And in case of OOM, die or just use `optarg` as `arg_verify_key` without expunging it? As you say, it’s snake oil, we *could* just proceed…